packages/core/src/database/index.ts

```diff
@@ -141,67 +141,69 @@ export const createDatabase = (args: {
   ////////
 
   let dialect: Database["dialect"];
-  let driver: Database["driver"];
+  let driver!: Database["driver"];
   let qb: Database["qb"];
 
   if (args.databaseConfig.kind === "pglite") {
     dialect = "pglite";
     namespace = "public";
 
-    const userFile = path.join(args.databaseConfig.directory, "public.db");
-    const syncFile = path.join(args.databaseConfig.directory, "ponder_sync.db");
+    const poolConfig: PoolConfig = {
+      max: 20,
+    };
 
-    // driver = {
-    //   user: createSqliteDatabase(userFile),
-    //   readonly: createReadonlySqliteDatabase(userFile),
-    //   sync: createSqliteDatabase(syncFile),
-    // };
+    driver = {
+      internal: createPool(poolConfig),
+      user: createPool(poolConfig),
+      readonly: createReadonlyPool(poolConfig),
+      sync: createPool(poolConfig),
+    };
```

Comment:
PGlite is single user/connection. And at the moment, it doesn't seem to support opening a database in readonly mode while another database is writing. So for now, we'll need to combine the `internal`, `readonly` and `user` drivers into one that we'll call `user` (those all touch the same tables). The sync can still be its own database and driver.

Here's what I'm thinking:

```ts
import { PGlite } from "pglite";

// ...

const userDir = path.join(args.databaseConfig.directory, "public");
const syncDir = path.join(args.databaseConfig.directory, "sync");

const userPglite = await PGlite.create(userDir)
const syncPglite = await PGlite.create(syncDir)

driver = {
  user: userPglite,
  sync: syncPglite,
}
```

The `driver` object should contain the low-level drivers so that we can safely tear them down them in the `kill` function.